src/framework/mlt_slices.h

```diff
@@ -38,4 +39,12 @@ extern int mlt_slices_count( mlt_slices ctx );
 
 extern mlt_slices mlt_slices_init_pool( int threads, int policy, int priority, const char* name );
 
+extern mlt_slices mlt_slices_get_global( mlt_schedule_policy policy );
```

Comment:
> why to rename: pool_lock -> g_lock

Because it was no longer specific to the properties pool, and I like to have a g_ prefix on global names.